dll/win32/shell32/lang/de-DE.rc

```diff
@@ -843,8 +843,8 @@ BEGIN
     IDS_FORMAT_COMPLETE "Formatieren abgeschlossen."
 
     /* Warning format system drive dialog strings */
-    IDS_NO_FORMAT_TITLE "Cannot format this volume"
-    IDS_NO_FORMAT "This volume cannot be formatted! It contains important system files in order for ReactOS to run."
+    IDS_NO_FORMAT_TITLE "Formatieren des Laufwerks nicht möglich"
+    IDS_NO_FORMAT "Dieses Laufwerk kann nicht formatiert werden! \nEs enthält wichtige Systemdateien, welche zur Ausführung von ReactOS benötigt werden."
```

Comment:
There is an unintended space in the string before the newline.

Reply:
fixed